backbone/src/main/java/org/researchstack/backbone/ResourceManager.java

```diff
@@ -113,4 +122,53 @@ public static ResourceManager getInstance() {
      * Onboarding sections
      */
     public abstract Resource getOnboardingManager();
+
+    /**
+     * The NameJsonProvider is a useful class that utilizes the
+     * ResourceManager to convert simple String names to Json Strings
+     */
+    public static class NameJsonProvider implements ResourceNameToStringConverter {
+
+        private Context context;
+
+        public NameJsonProvider(Context context) {
+            this.context = context;
+        }
+
+        @Override
+        public String getJsonStringForResourceName(String resourceName) {
```

Comment:
Looks like getJsonStringForResourceName and getHtmlStringForResourceName both handle any sort of string resource, not just JSON. 

Does this exist basically to work around needing to implement and instantiate ResourcePathManager.Resource? IMO it'd be worth simplifying ResourcePathManager, the implementation of typing is incomplete and feels cludgy (e.g. HTML vs HTM file extension, Resource.types are static variables, no extension/inheritance) and don't seem to add much value. 

Thoughts?